codegen-core/src/main/kotlin/io/vrap/rmf/codegen/types/PackageProvider.kt

```diff
@@ -67,11 +66,11 @@ class PackageProvider @Inject constructor(
 
     private inner class ResourcePackageSwitch : ResourcesSwitch<String>() {
         override fun caseMethod(`object`: Method): String {
-            return "$clientPackage.resource"
+            return "$clientPackage"
         }
 
         override fun caseResource(`object`: Resource?): String {
-            return "$clientPackage.resource"
+            return "$clientPackage"
```

Comment:
This change was required because it broke our current code, don't know why we added this in the first place.